server/data/hmppsAuthClient.test.ts

```diff
@@ -63,9 +63,9 @@ describe('hmppsAuthClient', () => {
             email: 'user@example.com',
             firstName: 'Auth',
             lastName: 'User',
-            locked: true,
-            enabled: false,
-            verified: false,
+            locked: false,
```

Comment:
I think it'd be a good idea now we have this filtering behaviour to add a couple more users to this response (possibly a locked/unverified one first) so we can check we're selecting the right one. Then we could update the `it` to say something like "should return the first matching **active** user from the API response"

Reply:
done